libraries/Microsoft.Bot.Connector/OAuthClient.cs

```diff
@@ -0,0 +1,326 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Bot.Connector
+{
+    using Microsoft.Rest;
+    using Microsoft.Rest.Serialization;
+    using Microsoft.Bot.Schema;
+    using Newtonsoft.Json;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Net;
+    using System.Net.Http;
+
+    public partial class OAuthClient : ServiceClient<OAuthClient>, IOAuthClient
```

Comment:
I don't understand the relationship between this class, the "Old" version of this class, and the Interface. 

My understanding is that this is a *Breaking Change*, which we can't do. I believe this needs to be OAuthClient2, and the old one can have a "[Obsolete / Depricated] attribute added to it.